packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.jsx

```diff
@@ -297,13 +301,15 @@ const FinancialAssessment = ({
 
     const getScrollOffset = () => {
         if (isMobile()) return is_appstore ? '160px' : '200px';
+        if (isMobile()) return is_mf && is_financial_information_incomplete && !is_submit_success ? '40px' : '80px';
```

Comment:
This is the same condition as the previous one... It will never be executed if the first one does